### ISSUE TITLE
It's very kind of you to offer a more clear version of the score_sde_torch.

### ISSUE THREAD

**Issue by yuanbingyan** [dexin-peng/score_sde_pytorch#1]
I am also a score-based diffusion model researcher. It's very kind of you to provide a clear and more abstract code. When I try to set the environment, I find there is something could be more detailed in the docker file, such as "your public key", which is easy to be ignored.
Besides, are you interested in the topic "solving inverse problem through score-based diffusion model"? If so, we can share the torch-version code to each other. Thanks for your attention.

**Comment by dexin-peng** [comment]
This repo is extracted from my repo in school. The original dockerfile is quite complicated, containing many other unnecessary components for the public, but I believe the ssh service works for everyone. When building the image for yourself, the .pub key should be replaced.
Currently I'm working on applying this model to some financial applications, inverse problems are on my future list. I've had wrote some classfier-guided code(given label, generate the image conditioning on the label, this requires training another conditioned neural nets). I think classfier-guided is the most simple case. Your insights should help me a lot. We can explore this further through email. Thanks for your attention as well.
